modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/bookmark/BookmarkHelper.kt

```diff
@@ -123,5 +123,5 @@ private fun buildPendingIntent(
     context,
     0,
     intent,
-    PendingIntent.FLAG_UPDATE_CURRENT.or(PendingIntent.FLAG_IMMUTABLE),
+    PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE,
```

Comment:
I don't quite understand why this is fixing the lint issue as I thought they were the same. 
<img width="933" alt="Screenshot 2024-09-13 at 9 30 21 PM" src="https://github.com/user-attachments/assets/ec607427-2800-4938-ac2d-ef5b12676d35">

Reply:
Looks like a bug in Lint. I forwarded it here: https://issuetracker.google.com/issues/348166278